config/v12/typo3-120.php

```diff
@@ -20,4 +20,5 @@
         ]);
     $rectorConfig->rule(\Ssch\TYPO3Rector\Rector\v12\v0\ReplacePreviewUrlMethodRector::class);
     $rectorConfig->rule(RenameExtTypoScriptFilesFileRector::class);
+    $rectorConfig->rule(\Ssch\TYPO3Rector\Rector\v12\v0\UseEmailLinkBuilderRector::class);
```

Comment:
Here is still the old name

Reply:
ah this is wrong either way as the \typo3\ is missing in the namespace.. that's some structural issue with the typo3-generate i reported already..

Reply:
The wrong namespace should be fixed with latest version